### PR TITLE
Implement public ObjectExt::get_property_type()

### PR DESCRIPTION
**Pull request by sdroege**
It was private before.

### CODE REVIEW

File: src/object.rs [hunk 1]
```diff
@@ -597,6 +585,22 @@ impl<T: IsA<Object> + SetValue> ObjectExt for T {
         }
     }
 
+
```
Comment (EPashkin) [inline src/object.rs]: Extra line

**Comment by sdroege** [comment]
Use case for this is that I need to get the type of flags/enums that are only defined in a plugin. And then use the GEnum/GFlags API to do something meaningful with them (patches for that following in a bit).

**Comment by EPashkin** [comment]
👍 
Thanks, @sdroege

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**